### PR TITLE
7.2: Create deployment and reference subfolders & fix .METADATA files

### PR DESCRIPTION
**Pull request by jhinkey**

### CODE REVIEW

**Comment by sez11a** [comment]
Just started reviewing :)

**Comment by sez11a** [comment]
Hey @jhinkey I don't understand the purpose of this PR. The .METADATA files should be ignored and should not appear in the repository. They're automatically generated and included in the Ant tasks from the `release-site.properties` file, AFAIK. Let's discuss tomorrow.

**Comment by jhinkey** [comment]
@sez11a - I reverted the .METADATA file related changes and fixed deployment/build.xml. Thanks!

File: deployment/build.xml [hunk 1]
```diff
@@ -5,7 +5,6 @@
 
 	<import file="../build-common.xml" />
 
-	<property name="purpose.dir" value="."/>
-	<property name="doc.dir" value="deployment"/>
+	<property name="purpose.dir" value="deployment"/>
```
Comment (sez11a) [inline deployment/build.xml]: How is hard-coding the dir name different from just supplying "."?

**Comment by jhinkey** [comment]
Hi @sez11a 

Using "." for purpose.dir was my mistake and doesn't work. 

The Edit on GitHub URLs follow this format:
`https://github.com/liferay/liferay-docs/blob/master/{purpose.dir}/{doc.dir}/{remaining-path-to-markdown-file}`

For example, the `build.xml` files provide these values for articles in `https://github.com/liferay/liferay-docs/blob/master/deployment/reference/`:

deployment/build.xml > purpose.dir = deployment
deployment/reference/build.xml > doc.dir = reference

**Pull request merged**